drivers/hmis/app/graphql/concerns/graphql_application_helper.rb

```diff
@@ -0,0 +1,24 @@
+module GraphqlApplicationHelper
+  extend ActiveSupport::Concern
+
+  def current_user
+    context[:current_user]
+  end
+
+  def hmis_user
+    Hmis::Hud::User.from_user(current_user)
+  end
```

Comment:
We often use `hmis_user` to talk about the Hmis::User, so I think this is pretty confusing. Can you call it `hud_user` or `current_hud_user`?

Reply:
Oh, I guess this was already there. I didn't realize.

Reply:
yeah, I just moved this code without changing it. I can take the opportunity to rename the method though, looks like there's only around 8 places where it's used